operations/csm-scheduled-tasks/.env.example

```
@@ -0,0 +1,44 @@
+# therefore this whole component) cannot function without these.
+OAUTH2_CLIENT_ID=
+OAUTH2_CLIENT_SECRET=
+OAUTH2_TOKEN_URL=
+
+# entity-service is this component's only durable state — every claim,
+# retry, success, and failure is persisted against its scheduled_task_run
+# table (see entity-service's own CLAUDE.md, "Scheduled task runs").
+# Authenticates with the shared OAUTH2_* credentials above, not its own —
+# only BaseURL/Scopes are specific to this client.
+ENTITY_SERVICE_BASE_URL=
+ENTITY_SERVICE_SCOPES= # comma-separated, optional
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)